app/app.go

```diff
@@ -623,6 +624,11 @@ func (app *BNBBeaconChain) initStaking() {
 		newCtx := ctx.WithSideChainKeyPrefix(storePrefix)
 		app.stakeKeeper.ClearUpSideVoteAddrs(newCtx)
 	})
+	upgrade.Mgr.RegisterBeginBlocker(sdk.FirstSunsetFork, func(ctx sdk.Context) {
+		chainId := sdk.ChainID(ServerContext.BscIbcChainId)
+		// enable channel but not send cross chain msg
+		app.scKeeper.SetChannelSendPermission(ctx, chainId, sTypes.StakeMigrationChannelID, sdk.ChannelAllow)
```

Comment:
You need set the fee of the new transaction type.

Reply:
Done